src/System.Net.Requests/src/Resources/Strings.resx

```diff
@@ -240,9 +240,6 @@
   <data name="net_PropertyNotSupportedException" xml:space="preserve">
     <value>This property is not supported by this class.</value>
   </data>
-  <data name="net_securityprotocolnotsupported" xml:space="preserve">
```

Comment:
This is the only resource whose value does not match the earlier instance in the file. The one here looks like it is a copy-paste error because the value doesn't make sense given the name. The value above is: `The requested security protocol is not supported.`